common/web/keyboard-processor/src/text/keyEvent.ts

```diff
@@ -10,18 +10,30 @@ import type Keyboard from "../keyboards/keyboard.js";
 import { type DeviceSpec } from "@keymanapp/web-utils";
 
 import Codes from './codes.js';
-import DefaultRules from './defaultRules.js';
 import { ActiveKeyBase } from "../index.js";
 
+interface DefaultRulesInterface {
+  forAny(Lkc: KeyEvent, isMnemonic: boolean): string;
+}
+
+export class BASE_DEFAULT_RULES {
```

Comment:
This feels backwards to have an interface with a global -- is it temporary infrastructure? I don't think it will fly long-term

Reply:
A changed it a bit, maybe that's better now?